packages/@lwc/template-compiler/src/parser/tag.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { BaseElement } from '../shared/types';
+import { TAGS_THAT_CANNOT_BE_PARSED_AS_TOP_LEVEL } from './constants';
+
+export function isNonTopLevelTag(element: BaseElement) {
```

Comment:
The name of this function is a little too generic. What about? 

```suggestion
export function isUnsafeSerializableElement(element: BaseElement) {
```

Reply:
Renamed to `isUnsafeTopLevelSerializableElement` for maximum clarity. 🙂